gradlew

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env sh
```

Comment:
This file, the properties file, and the core JAR file are created by running the command:

```gradle wrapper```

This allows people to use Gradle without having to install it first. More info:

https://docs.gradle.org/current/userguide/gradle_wrapper.htm

Once a project is gradlew-enabled, other users can run Gradle by executing `gradlew.bat` or `./gradle` (on MacOS and Linux) instead of the regular `gradle` command.